docs/manual.rst

```diff
@@ -1001,7 +998,7 @@ a schema of ``{'properties': {'bar': 'int', 'foo': 'str'}}`` will produce
 a shapefile where the first field is 'foo' and the second field is 'bar'. If
 you want 'bar' to be the first field, you must use a list of property items
 
-.. sourcecode:: python
+.. sourcecode:: pycon
```

Comment:
Changed to pycon as block contains pseudocode that would produce an error with rstcheck, the syntax of pycon seems not to be checked by rstcheck.